src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -238,6 +282,37 @@ export const WorkspaceDashboard = _.flow(
             storageCostEstimate
           ])
         ]),
+        div({ style: styles.header }, [
+          'Tags',
+          h(InfoBox, [
```

Comment:
I think we should move the icon so its in-line with the word "Tags" as it looks a little low right now. Also a bit of space between the icon and the word "Tags" would be good imo

Reply:
I agree, I was wondering if it was a little low. Glad you think so too!